.github/workflows/enforce-issue-template.yml

```diff
@@ -0,0 +1,35 @@
+name: enforce issue template
+on:
+  issues:
+    types: ['opened']
+jobs:
+  enforce-issue-template:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: "Look for a matching issue template"
+      shell: bash
+      run: |
+
```

Comment:
```suggestion
```

Reply:
Merp. The extra line was intentional. 🐭

Reply:
Eh, why? Looks confusing to me as it seems like an error.